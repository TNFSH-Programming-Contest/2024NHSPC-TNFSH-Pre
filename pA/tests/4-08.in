323138768812364143 733794618792202268 818893531203508896
818893531203508896 733794618792202268 323138768812364143
