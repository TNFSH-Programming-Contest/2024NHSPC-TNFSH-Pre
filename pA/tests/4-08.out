1705629093985462051
